internal/envoy/v3/route_test.go

```diff
@@ -423,33 +441,48 @@ func TestRouteRoute(t *testing.T) {
 				},
 			},
 		},
-		"mixed service w/ session affinity": {
```

Comment:
I removed this test b/c in practice we have a set of upstream clusters with homogenous load balancer policies (clusters are part of routes), but if anyone has context on why this might not be right, I would appreciate it